DS-Session-1/dataSructure/ArrayImplementationOfQueue.java

```diff
@@ -0,0 +1,100 @@
+package dataSructure;
+
+/**
+ * class for the array implementation of the Queue class
+ * @author Ishita_Mathur
+ *
+ */
+public class ArrayImplementationOfQueue implements Queue
+{
+	private int capacity;
+    	int queueArray[];
+    	int front = 0, rear = 0, currentSize = 0;
+    
+    	//constructor
+    	public ArrayImplementationOfQueue(int capacity)
+    	{
+        	this.capacity = capacity;
+        	queueArray = new int[this.capacity];
+    	}
+    
+    	/**
+     	 * method to add a value at the rear of the queue
+     	 * @param element to add
+     	 * @return AssertionError if queue is full
+     	 */
+	@Override
```

Comment:
Why `@Override` annotation? Class is just implementing the interface, no need to add the `@Override`.